README.md

```diff
@@ -84,14 +84,14 @@ Available options:
         Don't render the progress bar. default: false.
   -l/--latency
         Print all the latency data. default: false.
-  -I/--idReplacement
-        Enable replacement of [<id>] with a randomly generated ID within the request body. default: false.
```

Comment:
why are you removing this?

Reply:
It turns out this was actually missing from the `help.txt` file, and I copied the contents of that over during this PR. This is missing there too. I'll add it to that file, and fix this 👍